Core/src/org/sleuthkit/autopsy/communications/relationships/ContactDetailsPane.java

```diff
@@ -57,12 +76,24 @@ public void setNode(Node[] nodes) {
     public ExplorerManager getExplorerManager() {
         return explorerManager;
     }
-
-    @Override
-    public void setEnabled(boolean enabled) {
-        super.setEnabled(enabled);
-        nameLabel.setEnabled(enabled);
-        propertySheet.setEnabled(enabled);
+    
+    public ImageIcon getImageForFromArtifact(BlackboardArtifact artifact){
+        ImageIcon image = defaultImage;
+        
+        try {
+            for(Content content: artifact.getChildren()) {
+                if(content instanceof AbstractFile) {
```

Comment:
I assume the image is a child of the CONTACT artifact. But could there be more than one child that is a file?  Is there anything else we can check for to enure we returning the image?

Also, the loop should break after an image has been found?

Reply:
There could be more than one child, conceptually it could be something other than an image.  I will think about how to put a check.  If there is more than one image, I will put a break in so we get the first image.